src/VAC/VectorAnimationComplex/VAC.h

```diff
@@ -53,7 +54,7 @@ class KeyHalfedge;
 class PreviewKeyFace;
 class BoundingBox;
 
-class VAC: public SceneObject
+class Q_VPAINT_EXPORT VAC: public SceneObject
```

Comment:
we want the EXPORT on all classes in this folder, so we can access it from outside this library on Windows

Reply:
Understood. I will add EXPORT to all classes.